server/botserver.py

```diff
@@ -81,7 +81,8 @@ def parse_slack_message(self, message_list):
             if msg.get("type") == "message" and "subtype" not in msg:
                 if self.bot_at in msg.get("text", ""):
                     # Return text after the @ mention, whitespace removed
-                    return msg['text'].split(self.bot_at)[1].strip(), msg['channel'], msg['thread_ts'] if 'thread_ts' in msg else msg['ts'], msg['user']
+                    # leading ! removed
```

Comment:
looks good, just combine this with the comment one line above

Reply:
PEP8 requires 79 character max line length, but that's going to be a whole thing

updated